brainiak/hyperparamopt/norm.pyx

```diff
@@ -0,0 +1,60 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Cython file for optimizing GMM likelihood computation
+
+"""
+
+# Authors: Narayanan Sundaram (Intel Labs)
+
+#cython: embedsignature=True
+import numpy
+cimport numpy as np
+
+cdef extern from "math.h":
```

Comment:
It seems pretty ok to do. I was having problems earlier with pyximport import otherwise. This file is now gone as Cython was only really needed to get MCMC to run fast. This code has been replaced with scipy+numpy samplers.